gateway/sds_gateway/users/views.py

```diff
@@ -288,3 +300,298 @@ def get(self, request, *args, **kwargs) -> HttpResponse:
+                selected_files = []
+                selected_files_ids = request.POST.get("selected_files", "").split(",")
+                if selected_files_ids and selected_files_ids[0]:
+                    selected_files = list(
+                        File.objects.filter(uuid__in=selected_files_ids).values(
+                            "uuid",
+                            "name",
+                            "media_type",
+                            "directory",
+                            "size",
+                            "created_at",
+                        )
+                    )
+
+                return JsonResponse(
+                    {
+                        "success": True,
+                        "form": {
+                            "name": dataset_form.cleaned_data["name"],
+                            "author": dataset_form.cleaned_data["author"],
+                            "description": dataset_form.cleaned_data["description"],
+                        },
+                        "selectedCaptures": selected_captures,
+                        "selectedFiles": selected_files,
+                    }
+                )
+
+            # Process the dataset form for actual submission
+            dataset_form = DatasetInfoForm(request.POST, user=request.user)
+            if not dataset_form.is_valid():
+                return JsonResponse(
+                    {"success": False, "errors": dataset_form.errors}, status=400
+                )
+
+            # Get selected captures and files from hidden fields
+            selected_captures = request.POST.get("selected_captures", "").split(",")
+            selected_files = request.POST.get("selected_files", "").split(",")
+
+            # Validate that at least one capture or file is selected
+            if not selected_captures and not selected_files:
+                return JsonResponse(
+                    {
+                        "success": False,
+                        "errors": {
+                            "non_field_errors": [
+                                "Please select at least one capture or file."
+                            ]
+                        },
+                    },
+                    status=400,
+                )
+
+            # Create the dataset
+            dataset = Dataset.objects.create(
+                name=dataset_form.cleaned_data["name"],
+                description=dataset_form.cleaned_data["description"],
+                authors=[dataset_form.cleaned_data["author"]],
+                owner=request.user,
+            )
+
+            # Add selected captures to the dataset
+            if selected_captures:
+                captures = Capture.objects.filter(uuid__in=selected_captures)
+                dataset.captures.add(*captures)
+
+            # Add selected files to the dataset
+            if selected_files:
+                files = File.objects.filter(uuid__in=selected_files)
```

Comment:
This is causing an exception in QA.

When there are no files selected, `selected_files` here is a list with an empty string (`[""]`), and that gets evaluated to True:

```bash
python -c 'print(bool([""]))'

True
```

so the filter call ends up raising `ValidationError '“%(value)s” is not a valid UUID.'`